src/main/java/com/yurjinia/user/service/UserProfileService.java

```diff
@@ -60,7 +60,10 @@ public UserProfileDTO getUserProfile(String userEmail) {
     public void deleteAvatar(String userEmail) {
         UserProfileEntity userProfileEntity = getUserProfileByEmail(userEmail);
 
-        awss3Service.deleteImage(userProfileEntity.getAvatarId());
+        if (userProfileEntity.getAvatarId() == null) {
```

Comment:
move this if statement to separate method - call the method 'validateIfUserHasAvatar()'